web/src/assets/locales/de.json

```diff
@@ -77,6 +77,7 @@
         "cancel": "Abbrechen",
         "cancel_success": "Pipeline abgebrochen",
         "canceled": "Dieser Schritt wurde abgebrochen.",
+        "deploy": "Deployment starten",
```

Comment:
New translation keys must be added to en.json otherwise they will be removed by weblate and everyone expects to translate from English to other languages